apps/token-manager/app/src/components/LocalIdentityBadge/LocalIdentityBadge.js

```diff
@@ -9,9 +9,10 @@ const LocalIdentityBadge = ({ entity, ...props }) => {
   const network = useNetwork()
   const [label, showLocalIdentityModal] = useIdentity(entity)
   const handleClick = () => showLocalIdentityModal(entity)
+  
   return (
     <IdentityBadge
-      label={label || ''}
+      label={label || props.customLabel }
```

Comment:
What do you think about renaming this prop to something else, e.g. `defaultLabel` or `label`?

If we pick a different prop name than `label`, we should also put it in the prop types!